internal/plan/tarjan.go

```diff
@@ -38,8 +38,15 @@ func tarjanSort(successors map[string][]string) [][]string {
 		index:      make(map[string]int, len(successors)),
 	}
 
+	// Stabilize iteration through successors map to prevent
+	// disjointed components producing unstable output due to
+	// golang map randomized iteration.
+	stableIDs := make([]string, 0, len(successors))
 	for id := range successors {
-		id := string(string(id))
```

Comment:
That's a funny left over from an automatic replacement. The id used to be an ObjectId type.